3.analyze_data/README.md

```diff
@@ -0,0 +1,21 @@
+# Analyze predicted probability output
+
+In this module, we perform multiple analyses on the predicted probability data to validate the treatment (e.g., compound, CRISPR, or ORF) with the expected phenotypic class.
```

Comment:
```suggestion
In this module, we perform multiple analyses on the predicted probability data to validate the treatment (e.g., compound, CRISPR, or ORF).
```

I'm not sure I understand what "with the expected phenotypic class" means. Can you clarify? Or delete